$NetBSD$

Check for alloca(3) in order to do the proper dance in source codes
that uses it.

--- configure.ac.orig	2016-11-10 05:04:36.000000000 +0000
+++ configure.ac
@@ -74,6 +74,7 @@ AC_PROG_CXX
 AC_C_BIGENDIAN
 AC_HEADER_STDC
 AC_SYS_LARGEFILE
+AC_FUNC_ALLOCA
 AC_CHECK_LIB([m], [sin])
 AC_CHECK_FUNCS([log2f])
 AC_CHECK_PROGS([PYTHON2], [python2 python])
